configuration/globalproperties/clinical_gp.xml

```diff
@@ -1,5 +1,9 @@
 <config>
   <globalProperties>
+    <globalProperty>
+      <property>bahmni.encountersession.duration</property>
+      <value>240</value>
```

Comment:
I don't think we'll change this value. My understanding was that we rather use the Consultation tab to circumvent the issue brought by the short default 60m timeout.